docs/src/reference/contributing_to_sway.md

```diff
@@ -0,0 +1,99 @@
+# Contributing To Sway
+
+Thanks for your interest in contributing to Sway! This document outlines the process for installing and setting up the Sway toolchain, as well as some conventions on contributing to Sway.
+
+If you run into any difficulties getting started, you can always ask questions on our [Discord](https://discord.gg/xfpK4Pe).
+
+## Building and setting up a development workspace
+
+See the Sway book's [Introduction](../introduction/index.md) for instructions on installing and setting up the Sway toolchain..
+
+## Getting the repository
+
+```
+git clone https://github.com/FuelLabs/sway
```

Comment:
Should we be directing contributors to fork the sway repo first?

Reply:
Probably, yes